lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -695,6 +695,14 @@ function dosomething_reportback_set_files(&$entity, $values) {
     }
   }
 }
+function dosomething_reportback_views_pre_render(&$view) {
+  if ($view->name === 'reportback_files') {
+    $nid = $view->args[0];
+    $quantity = (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quanity') + (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quanity_override');
```

Comment:
This works for the `node/[nid]/rb` view.. but there is a taxonomy term view as well. http://staging.beta.dosomething.org/taxonomy/term/16/rb/reviewed

Possibly check path / `arg(0)` to only display while viewing from a node?